src/main/java/nextstep/ladder/domain/People.java

```diff
@@ -0,0 +1,16 @@
+package nextstep.ladder.domain;
+
+public class People {
+    private String name;
```

Comment:
```suggestion
    private final String name;
```
인스턴스 변수는 가능하면 final로 구현해 불변으로 구현하고, 값이 변경되는 경우만 final을 제거하는 습관을 들이는 것은 어떨까?